modules/serverless-logic/pages/cloud/build-and-deploy-with-serverless-operator-on-kubernetes.adoc

```diff
@@ -0,0 +1,193 @@
+[source,bash,subs="attributes+"]
+----
+kubectl create namespace kogito-workflows
+----
+
+=== Create a secret for the container registry authentication
+.Create a secret for the container registry authentication
+[source,bash,subs="attributes+"]
+----
+kubectl create secret docker-registry regcred --docker-server=<registry_url> --docker-username=<registry_username> --docker-password=<registry_password> --docker-email=<registry_email> -n kogito-workflows
+----
+
+or you can directly import your local docker config into your the Kubernetes cluster:
+
+.Create a secret for the container registry authentication based on local docker config
+[source,bash,subs="attributes+"]
+----
+kubectl create secret generic regcred --from-file=.dockerconfigjson=${HOME}/.docker/config.json --type=kubernetes.io/dockerconfigjson -n kogito-workflows
+----
+
+=== Create a {product_name} Serverless Platform containing the configuration (i.e. registry address, secret) for building your workflows
+
+The {product_name} Serverless Platform CR is the resource used to control the behavior of the {product_name} Serverless Operator.
+It defines the behavior of all Custom Resources (Workflow and Build) in the given namespace.
+
+Since the {product_name} Serverless Operator is installed in global mode, you will need to specify a {product_name} Serverless Platform CR in each namespace where you want the operator to be executed.
+You can find a basic {product_name} Serverless Platform CR example in the `config/samples` folder that you can simply apply to configure your operator.
+
+.Create a {product_name} Serverless Platform CR
+[source,bash,subs="attributes+"]
+----
+kubectl apply -f https://raw.githubusercontent.com/kiegroup/kogito-serverless-operator/{operator_version}/config/samples/sw.kogito_v1alpha08_kogitoserverlessplatform.yaml -n kogito-workflows
+----
+
+Note: In this Custom Resource, spec.platform.registry.secret is the name of the secret you created just before.
+
+[TIP]
+====
+You can also update "on-the-fly" the {product_name} Serverless Platform CR registry field with this command (change <YOUR_REGISTRY>)
+
+.Create a {product_name} Serverless Platform CR with a specific registry
+[source,bash,subs="attributes+"]
+----
+curl https://raw.githubusercontent.com/kiegroup/kogito-serverless-operator/{operator_version}/config/samples/sw.kogito_v1alpha08_kogitoserverlessplatform.yaml | sed "s|address: .*|address: <YOUR_REGISTRY>" | kubectl apply -f -
+----
+
+In order to retrieve the Cluster IP address of Minikube's internal registry to configure your platform, you can use the following command:
+
+.Retrieve Minikube registry internal IP
+[source,bash,subs="attributes+"]
+----
+kubectl get svc registry -n kube-system -ojsonpath='{.spec.clusterIP}'
+----
+====
+
+== Build and deploy your {context} application
+
+You can now send your {product_name} {context} Custom Resource to the operator which includes the {context} definition.
```

Comment:
```suggestion
You can now send your {product_name} Custom Resource to the operator which includes the {context} definition.
```

Reply:
Maybe we should have another attribute with product_name + context as its value so we don't have to do it every time we create prod docs.  Then we would need to edit just one value in the prod docs.